### PR TITLE
adding page

### PR DESCRIPTION
**Pull request by btaylor1989**

### CODE REVIEW

**Comment by RhuanKrum** [comment]
Hey there @btaylor1989 
Try to follow these steps in order to correct your code:
* Remove the "https://" from your URL
* Add a blank line in the end of your code

Good luck! 🤞

File: _pages/btaylor19892.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: btaylor1989
+pageUrl: https://btaylor1989.github.io/command-bt/
```
Comment (r5atom) [inline _pages/btaylor19892.yaml]: See if it works after you've removed `https://` from this line.

File: _pages/btaylor19892.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+timestamp: 2017-02-19
+---
```
Comment (r5atom) [inline _pages/btaylor19892.yaml]: This line should be followed by a "new line" by adding `enter` or `return` after `---`. 
This file should have 6 lines in total.

**Comment by brianamarie** [comment]
Hi @btaylor1989! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
Hi @btaylor1989!  I'm going to merge now. Nice work!

**Pull request merged**